tests/browser/index.test.js

```diff
@@ -310,6 +310,35 @@ suite('<Entity events/>', () => {
     });
   });
 
+  test('can remove event listener when component is detached', function (done) {
+    let bazHandlerCalled = false;
+    function bazHandler () { bazHandlerCalled = true }
+
+    ReactDOM.render(
+      <Scene>
+        <Entity id='foo'>
+          <Entity id='bar' events={{baz: bazHandler}}/>
+        </Entity>
+      </Scene>,
+      div
+    );
+    div.querySelector('a-scene').addEventListener('loaded', () => {
+      ReactDOM.render(
+        <Scene>
+          <Entity id='foo'/>
+        </Scene>,
+        div
+      );
+      div.querySelector('a-entity').emit('baz');
```

Comment:
I wish I knew how this could be. It seems like it's sort of keeping/caching the body of the event handler somewhere. It's not specific to this event handler or the `#bar` entity as it is occurs consistently when using a different entity/event handler.

Thanks!